### DISCUSSION TITLE
【添加功能】添加支持 UnblockNeteaseMusic

### DISCUSSION THREAD

**Discussion by MadisonWirtanen** [SPlayer-Dev/SPlayer#129] [想法 Ideas]
### 希望添加什么功能？

音乐解灰功能UnblockNeteaseMusic

### 具体信息

作者大大旧版本的SPlayer是支持解灰的，新版本会考虑加入相关功能吗
因为主要在手机上通过PWA使用，希望服务器部署时能加入解灰功能

**Comment by imsyy** [comment]
会有的

**Comment by imsyy** [comment]
目前已支持 酷我 / 咪咕 音源

![image](https://github.com/imsyy/SPlayer/assets/42232682/5877a39a-c0e3-4790-a6c5-5226874efbf9)

**Comment by Jeamorg** [comment]
似乎 web 端还未支持，而且需要自己搭建 api 接口

**Comment by imsyy** [comment]
> 似乎 web 端还未支持，而且需要自己搭建 api 接口

Web 端必须独立部署网易云 API 服务（因为直接从网易云请求数据会跨域限制，需要一层代理），解灰后续 Web 端应该会支持的
